addons/account/models/chart_template.py

```diff
@@ -299,6 +285,25 @@ def tax_template_changed(tax, template):
                 ('module', '=', 'account'),
             ]).unlink()
 
+        for model_name, records in data.items():
+            _fields = self.env[model_name]._fields
+            for xmlid, values in records.items():
+                x2manyfields = [
+                    fname
+                    for fname in values
+                    if fname in _fields
+                    and f"{model_name}.{fname}" not in custom_field
+                    and _fields[fname].type in ('one2many', 'many2many')
+                    and isinstance(values[fname], (list, tuple))
+                ]
+                if x2manyfields:
+                    rec = self.ref(xmlid, raise_if_not_found=False)
+                    if rec:
+                        for fname in x2manyfields:
+                            for i, (line, (command, _id, vals)) in enumerate(zip(rec[fname], values[fname])):
+                                if command == Command.CREATE:
+                                    values[fname][i] = Command.update(line.id, vals)
```

Comment:
I think we should add a comment above this `for model_name, records in data.items():` 
Like this for example
```
# converts `one2many` and `many2many` create command into update command
```
to avoid spending too much time trying to understand what it does while we don't really need to dive in the details